src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/global.yml

```diff
@@ -199,16 +202,27 @@ Resources:
                 - cloudformation.amazonaws.com
             Action:
               - sts:AssumeRole
+          - Effect: Allow
+            Sid: "AssumeRoleLambda"
+            Principal:
+              AWS:
+                - !Sub arn:${AWS::Partition}:iam::${DeploymentAccountId}:root
+            Action:
+              - sts:AssumeRole
+            Condition:
+              ArnEquals:
+                "aws:PrincipalArn": !Sub "arn:${AWS::Partition}:iam::${DeploymentAccountId}:role/adf-lambda-role"
```

Comment:
My mistake while merging, this is no longer in use. It got replaced by the the adf-automation roles instead.
```suggestion
```